todo_app/data/trello_items.py

```diff
@@ -0,0 +1,97 @@
+import os
+import requests
+import ssl
+from todo_app.data.item import Item
+from requests_kerberos import HTTPKerberosAuth, OPTIONAL
```

Comment:
I think this is an accidental auto-import, I can't see any evidence you use these impored functions, and they're not tracked in your dependencies, so best to remove this line!
```suggestion
```